lte/gateway/c/oai/lib/3gpp/3gpp_38.413.h

```diff
@@ -105,15 +178,14 @@ typedef struct PDU_Session_resource_setup_request_s {
   ran_ue_ngap_id_t
       ran_ue_ngap_id;  // This IE uniquely identifies the UE association over
                        // the NG interface within the NG-RAN node
-  Ngap_PagingPriority_ty RAN_Paging_Priority;
+  Ngap_PagingPriority_ty RAN_Paging_Priority; /*optional*/
   bstring nas_pdu;  // 5GC – UE or UE – 5GC message that is transferred without
-                    // interpretation in the NG-RAN node
+                    // interpretation in the NG-RAN node  /*optional*/
   // Ngap_ue_aggregate_maximum_bit_rate_t ue_aggregate_maximum_bit_rate;
```

Comment:
why is this commented out? should we remove it if there's no usage at the moment?

Reply:
It should be used eventually. @sanjay-ACL can you verify this?